internal/flagtypes/dryrun.go

```diff
@@ -0,0 +1,71 @@
+package flagtypes
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+)
+
+// ensure they conform to the interfaces
+var dryRun = DryRunNone
+var _ pflag.Value = &dryRun
+
+// DryRun is an enum flag for setting dry-run.
+type DryRun string
+
+const (
+	// DryRunNone disables dry-run. The build will be performed as usual
+	DryRunNone DryRun = "none"
+	// DryRunClient only logs what would be run, without contacting Kubernetes
+	DryRunClient DryRun = "client"
+	// DryRunServer submits server-side dry-run requests to Kubernetes
+	DryRunServer DryRun = "server"
+)
+
+// Set implements the pflag.Value and fmt.Stringer interfaces.
```

Comment:
What does "Set" mean here?

Reply:
Set is the name of the function.

godoc comments should always begin with the name of the target, except for package documentation that should start with `Package ${package name} ...`

Reply:
Then that's a glitch! Didn't realize, too tunnelvisioned :)